WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/RequestStatusChangeEmailTemplate.java

```diff
@@ -44,52 +44,37 @@ protected void addCustomReplacements(Replacements replacements)
             return _notificationBean.getRequestStatus();
         });
 
-        _replacements.add(new ReplacementParam<String>("orderDate", String.class, "Order placed date")
-        {
-            @Override
-            public String getValue(Container c) {return _notificationBean == null ? null : _notificationBean.getOrderDate();}
-        });
+        replacements.add("orderDate", String.class, "Order placed date", ContentType.Plain,
```

Comment:
Looks like "created" got clobbered in the merge; should add it back above "orderDate"